addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -54,34 +53,31 @@ const NavbarLinkPopoverWidget = weWidgets.LinkPopoverWidget.extend({
     _onEditLinkClick(ev) {
         var self = this;
         var $menu = this.$target.find('[data-oe-id]');
-        var dialog = new contentMenu.MenuEntryDialog(this, {}, null, {
-            name: $menu.text(),
-            url: $menu.parent().attr('href'),
-        });
-        dialog.on('save', this, link => {
-            let websiteId;
-            this.trigger_up('context_get', {
-                callback: function (ctx) {
-                    websiteId = ctx['website_id'];
-                },
-            });
-            const data = {
-                id: $menu.data('oe-id'),
-                name: link.content,
-                url: link.url,
-            };
-            return this._rpc({
-                model: 'website.menu',
-                method: 'save',
-                args: [websiteId, {'data': [data]}],
-            }).then(function () {
-                self.options.wysiwyg.odooEditor.observerUnactive();
-                self.$target.attr('href', link.url);
-                $menu.text(link.content);
-                self.options.wysiwyg.odooEditor.observerActive();
-            });
+        this.call('dialog', 'add', MenuDialog, {
```

Comment:
I thought that it would work but it will actually call the legacy dialog service, not the new one... I'm trying to see if there is a way to open an owl Dialog directly from the legacy widget but it appears that we have to go through a legacy component first

Reply:
I kept the code in even if it doesn't work as leaving the old one would prevent de module to load, thus preventing the entire website options to load. I will look back into after the merge, this code is isolated enough that I don't think it will disturbed someone else, but let me know what you think.